src/components/existing-request/existing-request-display.vue

```diff
@@ -44,8 +44,31 @@
                 <v-col cols="12" class="request-status">
                   <span>Request Status:</span>
                   &nbsp;
-                  <span :class="isNotPaid ? 'app-red' : isPaymentProcessing ? 'app-green' : ''">
-                    {{ requestStatusText }}</span>
+                  <span
+                    :class="isNotPaid ? 'app-red' : isPaymentProcessing ? 'app-green' : ''">
```

Comment:
If the alterate class binding syntax doesn't make this any neater, consider using a getter in the script instead.

Reply:
I created a getter to define what class to use here. Thanks.